### ISSUE TITLE
Can text be aligned vertically ?

### ISSUE THREAD

**Issue by danmo** [Ziv-Barber/officegen#119]

**Comment by njbotkin** [comment]
Yes, but how isn't documented.  Use `bodyProp` on text options, like so:

```js
slide.addText(`This text is aligned vertically`,
	{
		cx: `90%`,
		cy: `90%`,
		x: `5%`,
		y: `5%`,
		font_size: 58,
		bodyProp: {
			anchor: `ctr`,
		},
	})
```

The value of `bodyProp.anchor` is injected directly into the XML, so its enums are defined in the Open XML spec: https://docs.microsoft.com/en-us/dotnet/api/documentformat.openxml.drawing.textanchoringtypevalues?view=openxml-2.8.1

To see what other `bodyProp` officegen respects, look at https://github.com/Ziv-Barber/officegen/blob/87dd5501968501555ea3bee60043affc9f67c1fa/lib/pptx/genpptx.js#L1096-L1199

Open XML BodyProperty docs: https://docs.microsoft.com/en-us/dotnet/api/documentformat.openxml.drawing.bodyproperties?view=openxml-2.8.1

**Comment by njbotkin** [comment]
Whoops, this was already documented, but I missed it: https://github.com/Ziv-Barber/officegen/issues/42#issue-52970644
